docs/ref/forms/fields.txt

```diff
@@ -369,6 +369,11 @@ For each field, we describe the default widget used if you don't specify
     If provided, these arguments ensure that the string is at most or at least
     the given length.
 
+    .. attribute:: strip
+
+    If ``True`` (default) the value will be stripped of leading/trailing
```

Comment:
I was following the indentation of the previous block... why is this one different?